book/controller.rst

```diff
@@ -778,6 +778,20 @@ There are also special classes to make certain kinds of responses easier:
   :class:`Symfony\\Component\\HttpFoundation\\StreamedResponse`.
   See :ref:`streaming-response`.
 
+JSON helper
+~~~~~~~~~~~
+
+You can simplify sending JSON response with
+:method:`Symfony\\Bundle\\FrameworkBundle\\Controller\\Controller::json`
+helper::
```

Comment:
I miss here some information about the argument/arguments expected by this method.